src/transformers/models/clipseg/test.py

```diff
@@ -0,0 +1,58 @@
+import torch
```

Comment:
Yes I'll remove this script

Reply:
If you know it shouldn't be in the PR, why add it? 🤷‍♂️